ReactAndroid/src/main/java/com/facebook/react/common/ViewMethodsUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.facebook.react.common;
+
+import android.view.View;
+
+/**
+ * Various utility methods for {@link View}.
+ */
```

Comment:
Comment repeats the name of the class method without adding info. Please delete.